src/pybind/ceph_daemon.py

```diff
@@ -187,7 +188,8 @@ def _print_headers(self, ostr):
 
         sub_header = ""
         for section_name, names in self._stats.items():
-            for stat_name, stat_nick in names.items():
+            for stat_name in self._stats_order[section_name]:
```

Comment:
Don't the headers also have to follow this order?  And couldn't we get this effect by making the existing _stats an OrderedDict as well, and not need _stats_order?